src/API/OnboardingThemes.php

```diff
@@ -168,10 +168,10 @@ public function activate_theme( $request ) {
 
 		require_once ABSPATH . 'wp-admin/includes/theme.php';
 
-		$slug             = sanitize_key( $theme );
+		$slug             = sanitize_text_field( $theme );
```

Comment:
If we are going to allow case, I think we should not sanitize twice and instead replace the above instance of:

`$theme          = sanitize_title_with_dashes( $request['theme'] );`

With sanitization that allows case:

`$theme          = sanitize_text_field( $request['theme'] );`